cli.js

```diff
@@ -21,6 +21,10 @@ if (!args.dir || (!args.all && (!args.platform || !args.arch))) {
   process.exit(1)
 }
 
+if (args.tmpdir === 'false') {
+  args.tmpdir = false
+}
```

Comment:
I'd rather handle this by setting the `boolean` flag for this argument in the `minimist` options (cli.js)